doc/deployment_examples/sinfonier/feeding_sinfonier.md

```diff
@@ -0,0 +1,408 @@
+```
+An own terminal is required for this service. Another way is using `nohup` at the start and `&`at the end for running the server in background (Use restricted when brokers are running properly and if you know how `nohup` works)
+```
+nohup bin/zookeeper-server-start.sh config/zookeeper.properties &
+```
+
+[Top](#top)
+
+###<a name="section5.2"></a>Brokers
+####<a name=”section5.2.1></a>Configuration
+`Brokers` are used to distribute the information stored in `Kafka` and you can use one or several of them. This example use three `Brokers` and its configuration is described in this section. `Brokers` are implemented into `Zookeeper`, each one with its own port, i.e, a `Broker` with port 9092, another broker with port 9093, etc.
+A single `Broker` is configured through the following parameters:
+
+| Parameter | Mandatory | Value | Comments |
+|---|---|---|---|
+| broker.id | yes | 1 | An unique integer that define a broker |
+| port | yes | 9092 |  Port assigned to broker 1 |
+| host.name | yes | 0.0.0.0 |  For local connections must be `localhost`. For remote connections `0.0.0.0` allow them |
+| zookeeper.connect | yes | localhost:2181 | Usually zookeeper is running on a local machine. In other case, remote ip access is required with port 2181 |
+| log.dirs | yes | /tmp/kafka-logs-1 | kafka-logs-x, being x the broker.id |
+| advertised.host.name | no | your_remote_ip |  For remote connections of a known ip that allow it to access to the `broker` |
+| advertised.port | no | 9092 | For use a remote port different from the port broker set previously |
+
+A “multibroker” configuration can be configured creating different files, one by `Broker`, and setting same parameters like first, but changing `broker.id`, `port` and `logs.dir`.
+
+`Brokers` configurations must be stored in files like `serverx.properties`, being "x" the `broker.id` of each `Broker`.
+
+####<a name=”section5.2.2></a>Running
+Run a server is possible with a command:
+```
+bin/kafka-server-start.sh config/server1.properties
+```
+In case of “multibroker” cluster:
+```
+bin/kafka-server-start.sh config/server1.properties
+bin/kafka-server-start.sh config/server2.properties
+bin/kafka-server-start.sh config/server3.properties
+```
+Differents terminals are required for each server. Another way is using `nohup` at the start and `&`at the end for running the server in background (Use restricted when brokers are running properly and if you know how `nohup` works)
+```
+nohup bin/kafka-server-start.sh config/server1.properties &
+```
+
+[Top](#top)
+
+##<a name="section6"></a>Sinfonier
+Finally we reach the last element of our architecture: The consumer of the stored data. `Sinfonier` works as consumer, asking `Kafka` for information translated from `Orion`.
+
+`Kafka` use producers for create new messages into the `topics` and a consumer for show them. We have changed the roles in this architecture, making `Cygnus` work as producer and `Sinfonier` as a consumer.
```

Comment:
Kafka works as a queue, receiving data from the producers and sending it to the consumers. Regarding this particular architecture, Cygnus works as a producer, while sinfonier works as a consumer.